apps/approval/tests.py

```diff
@@ -63,3 +64,18 @@ def testIsFOSApplication(self):
         self.assertEqual(self.approval.is_fos_application(), False)
         self.approval.new_expiry_date = None
         self.assertEqual(self.approval.is_fos_application(), False)
+
+class EmailTest(TestCase):
```

Comment:
This test tests that Django's own mail function works. Django already tests that.

What you should do to test this behaviour is to test the view and send some data to it to verify that after someone clicks Accept/Deny, a mail is sent.